src/pages/AdminConsole.js

```diff
@@ -163,7 +163,7 @@ class AdminConsole extends Component {
             div({ className: 'row fsi-row-lg-level fsi-row-md-level no-margin' }, [
               // TODO: `admin_manage_access` will be fully deprecated with MDV
               div({
-                isRendered: (this.props.env === 'prod'),
+                isRendered: this.state.showAddUserModal,
```

Comment:
This div should be removed altogether as it is referencing an outdated page, Manage DARs.